### PR TITLE
feature/update-packages

### PR DESCRIPTION
**Pull request by PCOffline**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default {
       entityId: env.get('METADATA_ENTITY_ID').default('http://localhost:8000/metadata.xml').asUrlString(),
     },
     saml: {
-      authnContext: env.get('SAML_AUTH_CONTEXT').default('urn:oasis:names:tc:SAML:2.0:ac:classes:PasswordProtectedTransport').asString(),
+      authnContext: env.get('SAML_AUTH_CONTEXT').default('urn:oasis:names:tc:SAML:2.0:ac:classes:PasswordProtectedTransport').asArray(),
```
Comment (carmeltsadok) [inline src/config.ts:10]: Why

File: package.json [hunk 2]
```diff
@@ -15,41 +15,40 @@
   "license": "ISC",
   "dependencies": {
     "axios": "^0.21.1",
-    "body-parser": "^1.19.0",
     "cookie-parser": "^1.4.5",
-    "env-var": "^7.0.0",
+    "env-var": "^7.0.1",
```
Comment (carmeltsadok) [inline package.json:19]: There is still an error in git hub. Just writing it here

File: src/config.ts (hunk 1 above)
Comment (PCOffline) [reply src/config.ts:10]: Fixes a typescript error - it expects an array, not a string

File: package.json (hunk 2 above)
Comment (PCOffline) [reply package.json:19]: Done

**Review by carmeltsadok — APPROVED** [review]

**Pull request merged**